tests/storage/cbt/conftest.py

```
@@ -280,75 +238,136 @@ def pull_mode_token_secret(
-        client=unprivileged_client,
-        token_secret_ref=pull_mode_token_secret.name,
-        pvc_name=pull_backup_staging_pvc.name,
-        force_full_backup=False,
-        source=backup_tracker_source,
-    ) as backup:
-        wait_for_pull_backup_export_ready(backup=backup)
-        yield backup
+    incremental_count = request.param["incremental_count"]
+    completed_backups = []
+
+    def _deploy_pull_backup(name: str, force_full_backup: bool) -> VirtualMachineBackup:
```

Ruff (0.16.3)

[warning] Boolean-typed positional argument in function definition

(FBT001)